packages/genesys-spark-components/src/components/beta/gux-phone-input/gux-phone-input.tsx

```diff
@@ -515,8 +533,9 @@ export class GuxPhoneInput {
           }}
           type="tel"
           placeholder={this.phoneUtil.format(
-            this.phoneUtil.getExampleNumber(
-              this.region?.alpha2Code || this.defaultRegionCode || 'US'
+            this.phoneUtil.getExampleNumberForType(
+              this.region?.alpha2Code || this.defaultRegionCode || 'US',
+              this.parsePhoneNumberType(this.phoneNumberType)
```

Comment:
Instead of parsing the phoneNumberType here, we probably should have a setter that watches the property similar to `setDisplayFormat` (which sets an internal property that we can reference repeatedly).

Reply:
:+1: thanks for the review. Yes that makes sense. Here is the commit that adds the watch: 
https://github.com/MyPureCloud/genesys-spark/pull/577/commits/0193a03fa6c423c10cb0366f22273d08e4e849b3